packages/api/src/server.js

```diff
@@ -37,10 +40,18 @@ function errorHandler (err, req, reply) {
 let client
 let knex
 let fastify
+let dapi
+let dpp
 
 module.exports = {
   start: async () => {
     client = new Dash.Client()
+
+    await loadWasmDpp()
+    dpp = new DashPlatformProtocol()
+
+    dapi = new DAPI(DAPIConfig, dpp)
```

Comment:
`dapi = new DAPI(DAPIConfig, dpp)`

looks weird. either prepare DAPIClient instance before hand and pass it, or just initialize it in the constructor